src/Filters/DateRangeFilter.php

```diff
@@ -144,8 +144,6 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        $this->useColumn($this->getName());
-        $this->query(fn($query, $data) => $this->dateRangeQuery($query, $data));
```

Comment:
Why was this changed?

Reply:
Mmmmm, `useColumn` was overwriting the developer`s call to `useColumn` when configuring the filter. But I see there's an issue if the developer doesn't call `useColumn`, so I have to rethink this.

Let me check if there's another way and i'll update this PR with a better solution.

Reply:
I found the issue causing the developer's custom `column` to be overwritten. See commits below.

Reply:
Let me do a run test later today, then merge. 

Thanks for your efforts!!